app/src/main/java/com/w3engineers/unicef/telemesh/data/helper/RmDataHelper.java

```diff
@@ -617,6 +618,13 @@ private void sendToServer(List<BulletinTrackEntity> bulletinTrackEntities) {
 
     private BroadcastCommand getBroadcastCommand() {
         Payload payload = new Payload();
+
+        GeoLocation geoLocation = new GeoLocation()
+                .setLatitude("22.845272").setLongitude("89.531472");
```

Comment:
Static Geo location is set due to test the broadcast related things. during piloting to cox's bazar we need to make it dynamic

Reply:
We will work for updating geo location. But now we can use it statically for connecting dashboard and getting broadcast.